vaadin-spring-tests/test-ts-services/src/test/java/com/vaadin/flow/connect/OfflineFormIT.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.connect;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.UUID;
+
+import com.vaadin.flow.testutil.ChromeDeviceTest;
+import com.vaadin.testbench.TestBenchElement;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.mobile.NetworkConnection;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * Class for testing issues in a spring-boot container.
```

Comment:
BTW was this copied from somewhere else? Sounds a bit too generic for this use case. Not a blocker though.

Reply:
good catch, yep, it was a copy-paste, fixed